src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const rentCar = 40;
+  const minDaysForSmallDiscount = 3;
+  const maxDaysForSmallDiscount = 6;
+  const smallDiscount = 20;
+  const largeDiscount = 50;
+
+  if (days < minDaysForSmallDiscount) {
+    return days * rentCar;
```

Comment:
It would be great to create a variable for `days * rentCar` in order not to duplicate it three times